lms/djangoapps/ccx/views.py

```diff
@@ -612,6 +624,28 @@ def ccx_gradebook(request, course, ccx=None):
         })
 
 
+@transaction.non_atomic_requests
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@coach_dashboard
+def list_ready_downloads(request, course, ccx=None):
```

Comment:
Insted of copying & pasting, is there any way to re-use the code from the instructor dashboard?